src/ROUTES.ts

```diff
@@ -71,22 +71,26 @@ export default {
     SETTINGS_ABOUT: 'settings/about',
     SETTINGS_APP_DOWNLOAD_LINKS: 'settings/about/app-download-links',
     SETTINGS_WALLET: 'settings/wallet',
-    SETTINGS_WALLET_DOMAINCARDS: {
+    SETTINGS_WALLET_DOMAINCARD: {
```

Comment:
Hey, shouldn't the line below also be changed? I think you might forget changing that.

https://github.com/Expensify/App/blob/92813426a67df9b9a934d62dd9b1e4c41f5171a8/src/pages/settings/Wallet/ReportCardLostPage.js#L109C36-L109C63

cc @MrMuzyk, @narefyev91